src/DynamoCore/Logging/AnalyticsService.cs

```diff
@@ -84,6 +84,12 @@ internal static bool IsADPOptedIn
 
         internal static bool IsADPAvailable()
         {
+            if (Analytics.DisableAnalytics ||
+                adpAnalyticsUI == null)
+            {
+                return false;
+            }
+
             return adpAnalyticsUI.IsProviderAvailable();
```

Comment:
Was this the reason for the crash? Is `adpAnalyticsUI` `null `if `DisableAnalytics` is `true`?

Reply:
Yes. `adpAnalyticsUI` will be null when `DisableAnalytics` is true. 
We do not initialize `adpAnalyticsUI` so that no analytics assemblies will be loaded.